src/main/resources/liquibase/changelog/20191220183420_changelog.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+    <changeSet author="user (generated)" id="1576863265631-1">
+        <addColumn tableName="field_config">
+            <column name="unique_field" type="BIT"/>
+        </addColumn>
+    </changeSet>
+    <changeSet author="user (generated)" id="1576863265631-2">
+        <update tableName="field_config">
+            <column name="unique_field" valueBoolean="false"/>
+            <where>name='address'</where>
```

Comment:
I think if you remove the where condition it will set it for all of the fields, so you don't need to add separate updates for all of the fields